components/blocks/DirsToCardSections.vue

```diff
@@ -13,10 +13,10 @@
         <template>
           <h2
             class="section-title title has-text-dark"
-            :aria-label="'icon of' + category.fa.icon"
+            :aria-label="'icon of' + category.mdi.icon"
```

Comment:
I think we need a space after `icon of`

Reply:
changed this in some other parts of the code as well.